README.md

```diff
@@ -15,11 +15,16 @@ Open source tool and library for flashing Bouffalo RISC-V MCUs.
 # Supported Devices
 - [X] [Pinecil V2](https://wiki.pine64.org/wiki/Pinecil)
 
+# Supported OS
+- [x] Windows
+- [x] Linux
+- [ ] Apple (WIP: work-in-progress)
+
 # Building
 
 ## Clone repository
 
-If you have not cloned this repository locally; check out the git repository locally by running
+If you have not cloned this repository locally; copy the git repository locally by running
```

Comment:
`check out` or `clone` is correct term here, rather than `copy`

Reply:
I will change